openapi/openapi-client/src/main/openapi3/master_rest_v1.yaml

```diff
@@ -583,6 +583,13 @@ components:
         inNative:
           type: boolean
           description: Whether the thread is executing native code.
+        isDaemon:
```

Comment:
Remove `isDaemon` and `priority` and change `ThreadStack` with removing the fields.

Reply:
ok, done